### PR TITLE
Add files for code review from my trials

### PR DESCRIPTION
**Pull request by supreeti**
Code review form my trail projects

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+body {
```
Comment (supreeti) [inline style.css:1]: Use proper workflow and add linters to your code. It helps in:-
Catching syntax errors is more efficient. 
There is no need to debug simple mistakes like typos - the linter does it for you.

**Review by supreeti — COMMENTED** [review]
You can do it.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Good use of inline comments. ✔️
 - Good explanation of why to make changes. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] Your comment is good and explain why you ask for the changes. But your main comment does not look professional and does not inform the developer what the review is about like it needs to change something, it is approved, etc. Kindly make some changes in your main comment(you can get inspiration from this comment).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 - [OPTIONAL] You could use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖. Also you could format better the comment you request the change.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (supreeti) [reply style.css:1]: [Linters](https://github.com/microverseinc/linters-config/tree/master/html-css/.github/workflows)

**Review by supreeti — COMMENTED** [review]
Changes required for linters-config

1. Use linter.yml file 
2. Use proper github workflow

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @supreeti , 

I am Favour 👩🏽 , your reviewer for this PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- GitHub review feature used ✔️
- Inline comments used ✔️
- Stated reasons WHY a requested change needs to be implemented ✔️ 

## Optional suggestions

[OPTIONAL] 👍🏽 
- As suggested by @leonardodiasb, i think it will be great to always include a GIF to your review summary and some emoji's to your suggestions inorder to make them sound more human friendly 🤗 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by supreeti** [comment]
Thanks

**Pull request closed without merging**